GetRandomWordAPI/GetRandomWordAPI/Service/GetRandomWordService.cs

```diff
@@ -0,0 +1,22 @@
+using GetRandomWordAPI.Repository;
+
+namespace GetRandomWordAPI.Service;
+
+public class GetRandomWordService
```

Comment:
```suggestion
public class WordService
```

Reply:
We are naming this kind of services/repos/controllers after the entity it is working with. In this case it is a "word".